beginner_source/vt_tutorial.py

```diff
@@ -50,17 +50,16 @@
 # Follow the README at the DeiT repo for detailed information on how to
 # classify images using DeiT, or for a quick test, first install the
 # required packages:
-#
-# ::
-#
-#    pip install torch torchvision
-#    pip install timm
-#    pip install pandas
-#    pip install requests 
-#
-# then run the script below:
-#
 
+# pip install torch torchvision timm pandas requests 
+
+#######################################################
+# To run in Google Colab, uncomment the following line:
```

Comment:
This is good if we can't find a better solution that will generate slightly different format about `#! pip install...` when converting the source Python script to rst and ipynb.